workflows/gone.py

```diff
@@ -0,0 +1,74 @@
+"""
+Utilities for working with scm++
+"""
+import subprocess
+import tskit
+
+
+    # cmd = f"bgzip {path}.vcf"
+    # print("This is gunzip")
+    # print(cmd)
+    # logging.info("Running:" + cmd)
+    # subprocess.run(cmd, shell=True, check=True)
+
+
+    
+def params(gone_code):
+    """
+    modifies the GONE params file
+    """
+    phase=1 # 0 for pseudohaploid, 1 for unknown phase, 2 for phased
+    max_snps=500 # default=50000
```

Comment:
can't we have variables specifying this?

Reply:
Good idea you mean add these to the config file?

Reply:
sounds like a good plan to put this in the config file